### PR TITLE
Implement filter display UI of work tracker based on chris design 

### PR DESCRIPTION
**Pull request by lucaslyl**
linear: eco-93-display-filter-of-work-tracker-onselect

Screenshot 2024-11-19 at 17 29 41

### CODE REVIEW

File: packages/experiments-realm/app-task-card.gts [hunk 1]
```diff
@@ -349,59 +353,78 @@ class AppTaskCardIsolated extends Component<typeof AppCard> {
...
-          <div class='filter-display'>
+                {{item}}
+              </BoxelSelect>
+            {{/if}}
+          </div>
+
+          <div class='filter-display-sec'>
             {{#each this.filterTypes as |filterType|}}
+              {{log filterType}}
               {{#let (this.selectedFilterItems filterType) as |items|}}
```
Comment (tintinthong) [inline packages/experiments-realm/app-task-card.gts]: Remove log

File: packages/experiments-realm/app-task-card.gts [hunk 2]
```diff
@@ -349,59 +353,78 @@ class AppTaskCardIsolated extends Component<typeof AppCard> {
     return this.selectedItems.get(filterType) ?? [];
   }
 
+  @action getFilterIcon(filterType: FilterType) {
+    switch (filterType) {
```
Comment (tintinthong) [inline packages/experiments-realm/app-task-card.gts:357]: there is already a static configuration for all the filters. Let's keep all data in one place

**Comment by tintinthong** [comment]
Can u put the value for the filter value like Current Sprint slightly lighter

**Comment by lucaslyl** [comment]
> Can u put the value for the filter value like Current Sprint slightly lighter

Sure thank for the advise, but I had to tweak it a bit. I made the filter value more prominent so it stands out more than the key in the UI.

**Comment by tintinthong** [comment]
We have to make a decision here either to follow chris design and introduce those components in boxel-ui or perhaps these icons which I think match what he previously shared

```
import RectangleEllipsis from '@cardstack/boxel-icons/rectangle-ellipsis';
import User from '@cardstack/boxel-icons/user';
import FolderGit from '@cardstack/boxel-icons/folder-git';
```

I think lets use this for now

**Pull request merged**